packages/astro/test/fixtures/tsconfig-handling/baseUrl/tsconfig.json

```
@@ -0,0 +1,6 @@
+{
+  "files": ["i-have-base-url'"],
```

I know it's a fixture but that `'` would bother me 😆 
```suggestion
  "files": ["i-have-base-url"],
```